templates/common/.devcontainer/devcontainer.json/csharp/devcontainer.json

```diff
@@ -29,5 +29,8 @@
         3100
     ],
     "postCreateCommand": "",
-    "remoteUser": "vscode"
+    "remoteUser": "vscode",
+    "hostRequirements": {
+        "memory": "8gb"
```

Comment:
Is 8gb enough?

Reply:
Should we also look if cpu and storage is enough?

Reply:
Yes. With 8GB a user can launch both API and Web debugging scenarios in the same codespace instance and debug each app. Less than 8GB tends to not be sufficient in the presence of webpack.